packages/babel-plugin-emotion/src/utils/minify-utils.js

```diff
@@ -2,6 +2,11 @@
 // babel-plugin-styled-components
 // https://github.com/styled-components/babel-plugin-styled-components/blob/8d44acc36f067d60d4e09f9c22ff89695bc332d2/src/minify/index.js
 
+const CSS_ALLOWED_COMMENTS = [
+  // @noflip supports RTL via stylis plugin using cssjanus lib. If the comments
+  // that include this are stripped out, RTL will not work.
+  '@noflip'
+]
```

Comment:
Sorry, let me clarify, I meant any block comment that contains `@` anywhere in the comment, they should be kept so `/* @whatever */` and etc. should be kept.

Reply:
Sure, no problem. I can make that change.